src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Android/Airship.cs

```diff
@@ -377,5 +394,27 @@ public bool OnDeepLink(string deepLink)
             handler(deepLink);
             return true;
         }
+
+        public void OnChannelCreated(string channelId)
+        {
+            OnChannelCreation?.Invoke(this, new ChannelEventArgs(channelId));
+        }
+
+        public void OnChannelUpdated(string channelId)
+        {
+            OnChannelUpdate?.Invoke(this, new ChannelEventArgs(channelId));
+        }
+    }
+
+    public class ChannelEventArgs: EventArgs
```

Comment:
This class shouldn't be defined here, it needs to be accessible to both Android and iOS